shared/core/types/proto/relay.proto

```diff
@@ -18,12 +18,15 @@ message RelayPayload {
 }
 
 message RelayMeta {
+    reserved 5;
```

Comment:
No need to do this yet. We're not in production yet and this will undergo a lot of changes before we get there.

I use https://marketplace.visualstudio.com/items?itemName=ripwu.protobuf-helper to reset the numbers. When it starts mattering, we'll follow best practices.

Reply:
Thanks, fixed.